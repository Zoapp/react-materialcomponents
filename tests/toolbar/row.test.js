import React from "react";
import renderer from "react-test-renderer";
import Row from "libs/toolbar/row";

describe("toolbar/Row", () => {
  it("can render children", () => {
    const component = renderer.create(<Row>Children are here!</Row>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
