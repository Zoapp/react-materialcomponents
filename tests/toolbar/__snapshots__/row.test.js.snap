// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`toolbar/Row can render children 1`] = `
<div
  className="mdc-toolbar__row"
>
  Children are here!
</div>
`;
